src/bufr/BufrParser/BufrDescription.cpp

```diff
@@ -30,14 +24,13 @@ namespace
         const char* Mnemonics = "mnemonics";
         const char* Channels = "channels";
         const char* Exports = "exports";
-        const char* Datetime = "datetime";
-        const char* Mnemonic = "mnemonic";
```

Comment:
Could these deleted field break the downstream code? I managed to compile your branch but asking for posterity and my own understanding.

Reply:
These constants are not visible outside of BufrDescription as they are only declared in the .cpp file. So its not possible for them to affect downstream code (third party code that makes use of the library being generated by this code).